packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -124,7 +124,16 @@ describe('Incoming Payment Service', (): void => {
           walletAddressId,
           incomingAmount: {
             value: BigInt(123),
-            assetCode: asset.code.split('').reverse().join(''),
+            assetCode: String.fromCharCode(
+              ...asset.code.split('').map((letter) => {
+                let charCode = letter.charCodeAt(0)
+                if (charCode === 91) {
```

Comment:
I think this should be 90. I'm seeing 91 is `[` when I do `String.fromCharCode(91)` (so `ZZZ` becomes `[[[`). For whatever it's worth, it still works in that case but I think the intention is to keep it alphabetic.

Alternatively it looks like `assetCode: asset.code + 'x'` also works but I suppose its probably prudent to keep it a realistic assetCode